xcode/Subconscious/Shared/Config.swift

```diff
@@ -21,7 +21,7 @@ struct Config: Equatable, Codable {
     var pollingInterval: Double = 15
 
     /// Subsurface "orb" shader on main FAB
-    var orbShaderEnabled = true
+    var orbShaderEnabled = false
```

Comment:
Tempted to strip this config value and the if statement from the code, but it's not required to resolve the issue.

Reply:
Optional: we can strip.